app/assets/stylesheets/_category.scss

```diff
@@ -25,6 +25,10 @@
   text-decoration: none;
 }
 
+#cat-title{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space